### ISSUE TITLE
npower ERROR

### ISSUE THREAD

**Issue by moshouchajian** [lithammer/NeavUI#301]
This error occurs when my Death knight transfer from one area to another！

AddOns\nPower\core.lua:13: attempt to perform arithmetic on local 'start' (a nil value)
Count: 5

Call Stack:
[string "=[C]"]: ?
[string "@AddOns\nPower\core.lua"]:13: in function <AddOns\nPower\core.lua:11>
[string "@AddOns\nPower\core.lua"]:131: in function <AddOns\nPower\core.lua:116>

**Comment by L3n1n** [comment]
Same for me. This is bug or something like that. It can be fixed so easy. You need to replace this:

https://github.com/lithammer/NeavUI/blob/bd89bcae34bd5fc03d87cf9eac89f33bcdf53898/Interface/AddOns/nPower/core.lua#L11-L21

To this:

```lua
local function CalcRuneCooldown(num)
	if GetRuneCooldown(num) then
		local start, duration, runeReady = GetRuneCooldown(num)
		local time = floor(GetTime() - start)
		local cooldown = ceil(duration - time)

		if (runeReady or UnitIsDeadOrGhost("player")) then
			return "#"
		elseif (not UnitIsDeadOrGhost("player") and cooldown) then
			return cooldown
		end
	end
end
```
PS: As you can see i added additional condition for verification GetRuneCooldown()

**Comment by lithammer** [comment]
I pushed a commit that should hopefully fix this (see 454da6286b51d9706a7eb4f762e52a127013710b).
